spec/spec_helper.rb

```diff
@@ -1,7 +1,7 @@
 require 'simplecov'
 require 'simplecov-rcov'
 SimpleCov.formatter = SimpleCov::Formatter::RcovFormatter
-SimpleCov.minimum_coverage 100
```

Comment:
That's fine, this change doesn't prevent us from aiming to get back to 100% :) It just means we don't have a constantly failing test suite and invisibility of rubocop and brakeman failures!